Tests/LibWeb/Text/input/include.js

```diff
@@ -17,13 +18,21 @@ document.addEventListener("DOMContentLoaded", function () {
 });
 
 function test(f) {
+    if(__alreadyCalledTest)
+        throw new Error("test() may only be called once per page");
```

Comment:
You will need to run `prettier` on this file to format it (for example, like C++, we require a space between `if` and `(`). You will also want to put braces around the `if` block, other `prettier` will try to put it all on one line I think.

Reply:
Thanks, I hadn't got it installed properly so I ran it manually and then skipped the pre-commit... except I'd run it only on one of the files. :sweat_smile: